src/components/tracks/apply/_index.tsx

```diff
@@ -62,13 +49,42 @@ class Apply extends React.Component<IHeaderProps, {}> {
 
   render() {
     const { name, url } = this.props;
+    function getBatchDate(batchDate) {
```

Comment:
just fyi: this function could be outside of the render method, defined somewhere as a utility - even above the class would work. Otherwise it gets redefined on each "render"

Reply:
just did :)